elastic/examples/account_sample/src/ops/commands/put_bulk_accounts.rs

```diff
@@ -26,8 +26,8 @@ impl PutBulkAccounts for Client {
     }
 }
 
-fn put<B>(body: B) -> BulkRequest<'static>
-    where B: Into<Body<'static>>
+fn put<B>(body: B) -> BulkRequest<'static, B>
```

Comment:
This doesn't need to buffer the file into a `Vec`. It can just pass it straight on as the body.